### PR TITLE
Fix runtest-modules

### PR DESCRIPTION
**Pull request by nikromen**
#3431 was merged too fast :D

### CODE REVIEW

File: beaker-tests/Sanity/copr-cli-basic-operations/runtest-modules.sh [hunk 1]
```diff
@@ -142,7 +142,7 @@ rlJournalStart
         # $CHROOT variable
         rlRun "copr-cli create $PROJECT --chroot $PREV_CHROOT --chroot $CHROOT --chroot fedora-rawhide-i386"
         # move back to modules/testmodule once this is merged https://src.fedoraproject.org/modules/testmodule/pull-request/1
-        rlRun "copr-cli build-module --distgit fedora --url https://src.fedoraproject.org/fork/praiskup/modules/testmodule/raw/fix-rawhide/f/testmodule.yaml $PROJECT"
+        rlRun "copr-cli build-module --distgit fedora --url https://src.fedoraproject.org/fork/nikromen/modules/testmodule/raw/fix-rawhide/f/testmodule.yaml $PROJECT"
```
Comment (FrostyX) [inline beaker-tests/Sanity/copr-cli-basic-operations/runtest-modules.sh:145]: I'd have given you permissions to that fork but np.

**Comment by FrostyX** [comment]
Can you please write a commit description explaining why we cannot go to f41 yet? So that we can easily know in the future if the blocker is still there or not.

File: beaker-tests/Sanity/copr-cli-basic-operations/runtest-modules.sh (hunk 1 above)
Comment (nikromen) [reply beaker-tests/Sanity/copr-cli-basic-operations/runtest-modules.sh:145]: I was doing that at friday's night - I didn't want to bother you that late

**Comment by FrostyX** [comment]
Please keep the fedora version changes but revert back to those previous URLs. I updated the linked modules, they should build successfully now.

(I am hitting some issue with builders not starting, so I cannot confirm)

**Comment by nikromen** [comment]
let's migrate to f41 directly

**Pull request closed without merging**